systemtests/src/test/java/io/enmasse/systemtest/bases/web/WebConsoleTest.java

```diff
@@ -189,6 +189,42 @@ protected void doTestFilterAddressesByName() throws Exception {
                 String.format("Console failed, does not contain %d addresses", addressCount));
     }
 
+    protected void doTestDeleteFilteredAddress() throws Exception {
+        String testString = "addressName";
+        List<AddressWebItem> items;
+        int addressTotal = 2;
+
+        Destination destQueue = Destination.destination(AddressType.QUEUE, testString + "Queue",
+                getDefaultPlan(AddressType.QUEUE), Optional.empty());
+
+        Destination destTopic = Destination.destination(AddressType.TOPIC, testString + "Topic",
+                getDefaultPlan(AddressType.TOPIC), Optional.empty());
+
+        consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(sharedAddressSpace), addressApiClient,
+                sharedAddressSpace, defaultCredentials);
+        consoleWebPage.openWebConsolePage();
```

Comment:
you don't need to open web page if you then call createAdderrWebConsole which already contains this operation